OpenRA.Mods.Common/Traits/CommandBarBlacklist.cs

```diff
@@ -0,0 +1,27 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Blacklist certain order types to disable on the command bar when this unit is selected.")]
+	public class CommandBarBlacklistInfo : TraitInfo<CommandBarBlacklist>
```

Comment:
I don't consider this to be a blocker however, so it can wait for the TS commandbar PR.